README.md

```diff
@@ -13,6 +13,19 @@ Kubestr can assist in the following ways-
 
 [![asciicast](https://asciinema.org/a/7iJTbWKwdhPHNWYV00LIgx7gn.svg)](https://asciinema.org/a/7iJTbWKwdhPHNWYV00LIgx7gn)
 
+## Resources 
+Video 
+* [Cloud Native Live: Introducing Kubestr – A New Way to Explore your Kubernetes Storage Options](https://youtu.be/N79NY_0aO0w)
+* [Introducing Kubestr - A handy tool for Kubernetes Storage](https://youtu.be/U3Rt9vcuQdc)
+* [A new way to benchmark your kubernetes storage DoK Talks #71](https://www.youtube.com/watch?v=g64eIOk_Ob4)
+
+
+Blogs 
+* [Benchmarking and Evaluating Your Kubernetes Storage with Kubestr](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjpp6nn4NDyAhWXiVwKHaxHBigQFnoECAMQAQ&url=https%3A%2F%2Fblog.kasten.io%2Fbenchmarking-kubernetes-storage-with-kubestr&usg=AOvVaw1iH_P27EO2waz81znslF_Q)
```

Comment:
This URL appears funny, maybe it is a copy from a search result page instead of the proper page URL?